src/membership.md

```diff
@@ -36,18 +36,22 @@ Lang team members can propose new additions to the team as follows:
 These are the questions we ask ourselves when deciding whether someone would be a good choice as a lang team member.
 
 * Has this person demonstrated **strong language design skills**?
-    * Have they led an impactful initiative to completion?
-    * Have they identified a criticial flaw in someone else's design that led to the design being altered?
+    * Have they made major contributions to an area of the language, such as leading an impactful initiative to completion?
+    * Are they able to identify flaws in a design and, just as importantly, come up with creative solutions?
 * Is this person **responsible**?
     * When they agree to take on a task, do they either get it done or identify that they are not able to follow through and ask for help?
 * Is this person able to **lead others to a productive conversation**?
     * Are there times when a conversation was stalled out and this person was able to step in and get the design discussion back on track?
         * This could have been by suggesting a compromise, but it may also be by asking the right questions or encouraging the right tone.
-* Is this person able to **disagree productively**?
+* Is this person able to **disagree constructively and empathically**?
+    * The expectation is that team members go "above and beyond" the [Rust code of conduct](https://www.rust-lang.org/policies/code-of-conduct), embodying not only the letter but also the spirit.
     * When they are having a debate, do they make an active effort to understand and repeat back others' points of view?
-    * Do they "steelman", looking for ways to restate others' points in the most convincing way? 
```

Comment:
At first it bothered me to lose "steelman" here, but I think with the addition I made to the next point ("if they *or others* have a concern ..."), together with the addition above about collaborative disagreement, I think it's probably sufficiently captured without the jargon.

Reply:
Sounds good.